packages/flutter/lib/src/services/text_input.dart

```diff
@@ -98,7 +99,15 @@ class TextInputType {
   const TextInputType.numberWithOptions({
     this.signed = false,
     this.decimal = false,
-  }) : index = 2;
+  }) : index = 2, forceMultiline = null;
+
+  /// Optimize for none information.
+  ///
+  /// Requests a none keyboard with additional settings.
+  /// The [forceMultiline] parameter is optional.
```

Comment:
Consider adding a comment about how `forceMultiline` only applies in a web context.

Reply:
Also we should probably explain why someone might use this i.e. the custom text input control use-case.